src/bentoml/_internal/cloud/deployment.py

```diff
@@ -560,3 +565,98 @@ def terminate(
         if res is None:
             raise BentoMLException("Terminate deployment request failed")
         return res
+
+    @classmethod
+    def _create_deploymentV2(
+        cls,
+        create_deployment_schema: CreateDeploymentSchemaV2,
+        context: str | None = None,
+    ) -> DeploymentSchema:
+        cloud_rest_client = get_rest_api_client(context)
+        res = cloud_rest_client.create_deployment_v2(create_deployment_schema)
```

Comment:
nit: should we move this to v2 namespace?

```suggestion
        res = cloud_rest_client.v2.create_deployment(create_deployment_schema)
```

Reply:
I mean the rest_client is never a public API, so breaking is ok?